.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/SLF4Swift.iml" filepath="$PROJECT_DIR$/.idea/SLF4Swift.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
AppCode actually, but no, not useful. It's a local project file I didn't mean to commit.